pkg/manager/member/tikv_member_manager.go

```diff
@@ -201,6 +201,11 @@ func (tkmm *tikvMemberManager) syncStatefulSetForTidbCluster(tc *v1alpha1.TidbCl
 		return err
 	}
 
+	// Recover failed stores if any before generating desired statefulset
```

Comment:
We should also try to recover for TiFlash.

Reply:
done